src/styles/boxes/moyo_logo.scss

```diff
@@ -0,0 +1,4 @@
+.moyo-logo {
+  position: absolute;
+  top: 10px;
+}
```

Comment:
no need to make logo absolute, just add align-items: center to the nav to center all elements vertically